app/schemas/archive-format.json

```diff
@@ -0,0 +1,222 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://the-federation.info/specs/backup-restore#archive_format",
```

Comment:
What document is expected to be at the URL? Does it have to be something official? I'm writing something like an implementation plan&details for the account migration. But that though related it doesn't specify the exact format of the archive (but rather relies on the schema). Do I need to have a separate document defining the archive format? To me it seemed, that I could avoid having a document for the archive specification since there is a schema (it's kinda self-documenting).